examples/SharedMemory/PhysicsServerCommandProcessor.cpp

```diff
@@ -11209,7 +11209,8 @@ bool PhysicsServerCommandProcessor::processRemoveBodyCommand(const struct Shared
 				{
 					if (m_data->m_pluginManager.getRenderInterface())
 					{
-						m_data->m_pluginManager.getRenderInterface()->removeVisualShape(bodyHandle->m_multiBody->getBaseCollider()->getUserIndex3());
+                                                m_data->m_pluginManager.getRenderInterface()->removeVisualShape(bodyUniqueId);
```

Comment:
By my understanding this function call ends up with `EGLRendererVisualShapeConverter::removeVisualShape`. This function wants to have `collisionObjectUniqueId`, which I would assume is the `bodyUniqueId`.

In the master version, when running the simple test script, I was getting values of 1 from `getUserIndex3`, in cases where `collisionObjectUniqueId`,`m_internalInstanceIndex`, and `m_publicGraphicsInstances` are 0.

Reply:
In the code block below, which I can't place a comment next to, it iterates over all of the links, and removes the visual shapes (line 11226). This makes it look like `removeVisualShape` should not be getting `collisionObjectUniqueId` but an index of `m_visualShapes` instead.